playbooks/byo/openshift-cluster/config.yml

```diff
@@ -3,6 +3,19 @@
   tags:
   - always
 
+- name: Verify Requirements
+  hosts: OSEv3
```

Comment:
Nevermind, I see that you're selecting which checks to apply based on byo group names.